packages/graphql/src/schemaTypes/objectTypes/gql-Query.ts

```diff
@@ -101,6 +102,12 @@ export const Query = objectType({
       resolve: (source, args, ctx) => ctx.coreData.authState,
     })
 
+    t.field('studio', {
+      type: Studio,
+      description: 'Studio data',
```

Comment:
should be better describe what `studio` is here?

Reply:
we might not have answer right now either

Reply:
Yeah I'm not sure we have a exhaustive description yet. But I can provide something better.

Reply:
Added a little bit better description: [`634109a` (#31033)](https://github.com/cypress-io/cypress/pull/31033/commits/634109a70223aa69af77b1fa5415ce3891556128)